sdk/inc/azure/core/az_json.h

```diff
@@ -278,42 +283,43 @@ typedef struct
 /**
  * @brief Initializes an #az_json_writer which writes JSON text into a buffer.
  *
- * @param[out] json_writer A pointer to an #az_json_writer instance to initialize.
- * @param[in] destination_buffer An #az_span over the byte buffer where the JSON text is to be
- * written.
+ * @param[out] out_json_writer A pointer to an #az_json_writer instance to initialize.
+ * @param destination_buffer An #az_span over the byte buffer where the JSON text is to be written.
  * @param[in] options __[nullable]__ A reference to an #az_json_writer_options
  * structure which defines custom behavior of the #az_json_writer. If `NULL` is passed, the writer
- * will use the default options (i.e. #az_json_writer_options_default()).
+ * will use the default options (i.e. #az_json_writer_options_default).
```

Comment:
`az_json_writer_options_default` is a method.

```suggestion
 * will use the default options (i.e. #az_json_writer_options_default()).
```